packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -98,9 +98,17 @@ export class ColorPicker
    * When `true`, an empty color (`null`) will be allowed as a `value`.
    *
    * When `false`, a color value is enforced, and clearing the input or blurring will restore the last valid `value`.
+   *
+   * @deprecated Use `clearable` instead
    */
   @Prop({ reflect: true }) allowEmpty = false;
 
+  @Watch("allowEmpty")
+  @Watch("clearable")
+  handleAllowNullValues(): void {
+    this.isClearable = !!(this.clearable || this.allowEmpty);
```

Comment:
The `!!` is unnecessary as you are using two boolean properties with default values.